### DISCUSSION TITLE
Download Version 2.1.6 on sourceforge, but no related source code on github?

### DISCUSSION THREAD

**Discussion by Discussion-008** [dukus/digiCamControl#405] [General]
Hello dukus and others,

I have seen that there is a **stable version 2.1.6** for download on **sourceforge**, but the **version on github is 2.0.0**. I also do not see a label or release 2.1.6 on github. Is this repo still the location of the current sourcecode for digiCamControl, or is it no longer open source, or anything else? I am just curious to understand whether the 2.1.6 on sourceforge is really a valid version, or not.

**Comment by dukus** [comment]
The main branch is the latest release. But unfortunately isnt tagged.
